src/main/resources/datical/datical_status_detail.sh.ftl

```diff
@@ -10,4 +10,5 @@
 
 -->
```

Comment:
Is this script still used? I don't see it in the script based rule.

Reply:
You are correct. We no longer use "datical_status_detail.sh.ftl" file.